src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+/*
+ * Tagging object class, to record list of commands for tags used in session
+ */
+
+public class Tagging {
+
+	public static final String _ADD = "+"; 
+	public static final String _REMOVE = "-";
+	
+	public static ArrayList<String> tagHistory;
+	
+	private String command;
+	private Person person;
+	private Tag tag;
+	
+	public Tagging ( String command, Person person, Tag tag ){
+		
+		if ( command.equalsIgnoreCase("add"))
+			command = _ADD;
+		else
+			command = _REMOVE;
+		
+		this.person = person;
+		this.tag = tag;
+		
+		addToHistory();
+		
+	}
+	
+	public static void initializeHistory(){
+		
+		if(tagHistory == null)
+			tagHistory = new ArrayList<String>();
+			
+	}
+	
+	
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.